pkg/test/ginkgo/junit.go

```diff
@@ -76,6 +76,16 @@ type JUnitTestCase struct {
 	// Duration is the time taken in seconds to run the test
 	Duration float64 `xml:"time,attr"`
 
+	// Weight is the impact of a failing test on the suite.
```

Comment:
what should consumers assume when this is empty? Or does it even make sense to define that?

Reply:
It's `omitempty`, so it can only be unset.  It should never be unset for any failed tests.  I guess we could call that out in the docstring.  But it gets kind of complicated if test-output consumers are themselves effectively a subsequent test suite.  Would the job-detail page distinguish between test passed/failed and an orthogonal test output strictly valid, invalid but still mostly readable, or completely invalid (e.g. nominally JUnit content that is not XML)?